meta-lmp-base/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -40,6 +44,16 @@ PACKAGECONFIG ?= " \
     zstd \
 "
 
+PACKAGECONFIG[p11kit] = "-Dp11kit=true,-Dp11kit=false,p11-kit"
+PACKAGECONFIG[cryptsetup-plugins] = "-Dlibcryptsetup-plugins=true,-Dlibcryptsetup-plugins=false,cryptsetup,,cryptsetup"
+
```

Comment:
is it upstreamable?

Reply:
This can land upstream, yes (planning on pushing that in parallel).